CONTRIBUTING.rst

```diff
@@ -1,56 +1,54 @@
+============================
+Contribute to JDAT Notebooks
+============================
 
-   ``jdat_notebooks`` are one component of STScI's larger [Data Analysis Tools Ecosystem](https://jwst-docs.stsci.edu/jwst-post-pipeline-data-analysis).
+This document outlines the process for contributing code to the Data Analysis Tools Ecosystem by following the JWST Data Analysis Tools (JDAT) Approach to "Notebook-Driven Development". The procedures here outline the process for
+getting a notebook through successive development stages to become something that can be "live" on the spacetelescope notebooks repository.
 
-   The [jdat_notebooks repository](https://github.com/spacetelescope/jdat_notebooks) is for already-reviewed notebooks that are ready for consumption by the science community.  If you wish to contribute notebooks or technical expertise, we refer you to [dat_pyinthesky](https://github.com/spacetelescope/dat_pyinthesky/tree/master/jdat_notebooks) as the place to submit a first draft.  The description below outlines the whole process, which begins in `dat_pyinthesky` but finishes in `jdat_notebooks`.
-
-
-# Development Procedure for JDAT Notebooks
-
-This document is a description of the JWST Data Analysis Tools (JDAT) Approach to
-"Notebook-Driven Development".  The procedures here outline the process for
-getting a notebook through successive development stages to become something
-that can be "live" on the spacetelescope notebooks repository.
 
 These notebooks can have many varied science cases, but follow a relatively
-standard workflow:
+standard workflow
 
-1. Draft Stage
-2. Baseline Stage
-3. Notebook-driven development
-4. Advanced Stage
-5. Final Notebook - Maintenance Stage
-6. Revision based on Community feedback
+1. `Draft Stage`_
+2. `Baseline Stage`_
+3. `Notebook-driven development`_
+4. `Advanced Stage`_
+5. `Final Notebook - Maintenance Stage`_
+6. `Revision based on Community feedback`_
 
 These stages and the process for moving from one to the other are described below.
 
 **The procedure to submit the notebook via a Pull Request is described at the end of this document.
 This is repeated for each of the 5 stages.**
 
 Note that there is much more information on writing Jupyter notebooks at the
-[STScI notebook style guide](https://github.com/spacetelescope/style-guides/blob/master/guides/jupyter-notebooks.md),
+`STScI notebook style guide <https://github.com/spacetelescope/style-guides/blob/master/guides/jupyter-notebooks.md>`_,
 and similar guidance for Python code at the
-[STScI Python style guide](https://github.com/spacetelescope/style-guides/blob/master/guides/python.md).
+`STScI Python style guide <https://github.com/spacetelescope/style-guides/blob/master/guides/python.md>`_.
 These guidelines are in place to make review steps easier.
 
-## Draft Stage
+.. important::
+
+    Please note that all JDAT related code is written `Python 3`; `Python 2` is not supported.
```

Comment:
```suggestion
    Please note that all JDAT related code should be written in `Python 3`; `Python 2` is not supported.
```
This is a minor typo fix *but* it raises a larger question of whether we should specify a version more precisely. https://github.com/spacetelescope/style-guides/blob/master/guides/software-versioning.md has some guidelines about this, which we could either reference, or we can have a more precise answer for jdat.  That might be a distraction from the maint point of this PR though - if you like I can make a separate issue to address this later.